standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -368,7 +368,7 @@ public GetOpenTxnsInfoResponse getOpenTxnsInfo() throws MetaException {
          */
         dbConn = getDbConn(Connection.TRANSACTION_READ_COMMITTED);
         stmt = dbConn.createStatement();
-        String s = "select ntxn_next - 1 from NEXT_TXN_ID";
+        String s = "select \"NTXN_NEXT\" - 1 from \"NEXT_TXN_ID\"";
```

Comment:
You have not capitalized this. Is this intentional?

Reply:
no, that one just got away :)